app/scripts/lib/fxa-client.js

```diff
@@ -125,13 +125,32 @@ define(function (require, exports, module) {
         });
     },
 
+    confirmSignInResend: function (relier, sessionToken, options) {
+      options = options || {};
+
+      return this._getClient()
+        .then(function (client) {
+          var clientOptions = {
+            redirectTo: relier.get('redirectTo'),
+            service: relier.get('service')
+          };
+
+          if (options.resume) {
+            clientOptions.resume = options.resume;
+          }
+
+          return client.confirmSignInSendCode(sessionToken, clientOptions);
+        });
+    },
```

Comment:
I imagine we'll need an equivalent to actually send the verification code to the auth server to verify the token.